redeem/Redeem.py

```diff
@@ -283,8 +283,8 @@ def __init__(self, config_location="/etc/redeem"):
       adc = self.printer.config.get("Heaters", "path_adc_" + e)
       if not self.printer.config.has_option("Heaters", "sensor_" + e):
         sensor = self.printer.config.get("Heaters", "temp_chart_" + e)
-        logging.warning("Deprecated config option temp_chart_" + e + " use sensor_" + e +
```

Comment:
The format of this statement was in accordance with the specification.

Reply:
All I can say is that I ran `yapf` on this file along with Pipe.py and it made this change.

Reply:
I'm only going by the circleci report:

https://circleci.com/gh/Wackerbarth/workflows/redeem

Reply:
Then we should probably figure out what difference is causing this:
```
$ yapf --diff ./redeem/Redeem.py
--- ./redeem/Redeem.py  (original)
+++ ./redeem/Redeem.py  (reformatted)
@@ -280,8 +280,8 @@
       adc = self.printer.config.get("Heaters", "path_adc_" + e)
       if not self.printer.config.has_option("Heaters", "sensor_" + e):
         sensor = self.printer.config.get("Heaters", "temp_chart_" + e)
-        logging.warning("Deprecated config option temp_chart_" + e + " use sensor_" + e +
-                        " instead.")
+        logging.warning(
+            "Deprecated config option temp_chart_" + e + " use sensor_" + e + " instead.")
       else:
         sensor = self.printer.config.get("Heaters", "sensor_" + e)
       self.printer.thermistors[e] = TemperatureSensor(adc, 'MOSFET ' + e, sensor)
```

Reply:
```
$ yapf -v
yapf 0.20.1
```

Reply:
You have an older yapf
From the circleci run:
```Requirement already satisfied: yapf in ./venv/lib/python2.7/site-packages (0.24.0)```